plugin/trino-cassandra/src/test/java/io/trino/plugin/cassandra/TestCassandraConnectorTest.java

```diff
@@ -1247,6 +1248,10 @@ void testPartitioningKeys()
     @Test
     void testSelectClusteringMaterializedView()
     {
+        if (!hasBehavior(SUPPORTS_CREATE_MATERIALIZED_VIEW)){
+            abort("CREATE MATERIALIZED VIEW disabled");
+        }
```

Comment:
We shouldn't add this check. The connector already knows its behavior. Please revert.